esp-hal-common/src/gpio/esp32.rs

```diff
@@ -298,22 +298,22 @@ pub enum OutputSignal {
     VSPICS0                  = 68,
     VSPICS1                  = 69,
     VSPICS2                  = 70,
-    LEDC_HS_SIG_0            = 71,
-    LEDC_HS_SIG_1            = 72,
-    LEDC_HS_SIG_2            = 73,
-    LEDC_HS_SIG_3            = 74,
-    LEDC_HS_SIG_4            = 75,
-    LEDC_HS_SIG_5            = 76,
-    LEDC_HS_SIG_6            = 77,
-    LEDC_HS_SIG_7            = 78,
-    LEDC_LS_SIG_0            = 79,
-    LEDC_LS_SIG_1            = 80,
-    LEDC_LS_SIG_2            = 81,
-    LEDC_LS_SIG_3            = 82,
-    LEDC_LS_SIG_4            = 83,
-    LEDC_LS_SIG_5            = 84,
-    LEDC_LS_SIG_6            = 85,
-    LEDC_LS_SIG_7            = 86,
+    LEDC_HS_SIG0             = 71,
```

Comment:
What's the purpose of renaming these?

Reply:
I wanted to have consistent names for every chip.